File: front-end/src/app/service-models/app/v1/client/client-employee-attendance-extended-user-s-m.ts
import { ClientEmployeeAttendanceSM } from './client-employee-attendance-s-m';

export class ClientEmployeeAttendanceExtendedUserSM extends ClientEmployeeAttendanceSM {
    userName!: string;
    employeeCode!: string;
    presentCount!: number;
    absentCount!: number;
    leaveCount!: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/Client/ClientEmployeeAttendanceExtendedUserSM.cs
namespace Siffrum.Web.Payroll.ServiceModels.v1.Client
{
    public class ClientEmployeeAttendanceExtendedUserSM : ClientEmployeeAttendanceSM
    {
        public string UserName { get; set; }
        public string EmployeeCode { get; set; }
        public int PresentCount { get; set; }
        public int AbsentCount { get; set; }
        public int LeaveCount { get; set; }
    }
}
